fe_utils/quadrature.py

```diff
@@ -1,5 +1,5 @@
 from numpy.polynomial.legendre import leggauss
-from .reference_elements import ReferenceInterval, ReferenceTriangle
+from reference_elements import ReferenceInterval, ReferenceTriangle
```

Comment:
This is another code-breaking change. Why do this?

Reply:
I'm running files in PyCharm as opposed to running from a terminal. I've been getting errors from relative imports and absolute imports seem to fix them. By the time I submit.... I will return all the imports back to relative.